src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/util/LicenceEventType.kt

```diff
@@ -1,28 +1,30 @@
 package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util
 
-enum class LicenceEventType {
-  CREATED,
-  SUBMITTED,
-  BACK_IN_PROGRESS,
-  APPROVED,
-  ACTIVATED,
-  SUPERSEDED,
-  HARD_STOP_CREATED,
-  HARD_STOP_SUBMITTED,
-  HARD_STOP_REVIEWED_WITHOUT_VARIATION,
-  HARD_STOP_REVIEWED_WITH_VARIATION,
-  VARIATION_CREATED,
-  VARIATION_SUBMITTED_REASON,
-  VARIATION_IN_PROGRESS,
-  VARIATION_SUBMITTED,
-  VARIATION_REFERRED,
-  VARIATION_APPROVED,
-  INACTIVE,
-  RECALLED,
-  VERSION_CREATED,
-  NOT_STARTED,
-  TIMED_OUT,
-  HDC_CREATED,
-  HDC_VERSION_CREATED,
-  HDC_SUBMITTED,
+import com.fasterxml.jackson.annotation.JsonValue
+
+enum class LicenceEventType(@JsonValue val description: String) {
```

Comment:
Isn't this going to change the response type of other models? 
Think we need a SAR version of this - or potentially we can get rid of LicenceEvents entirely